src/core/core.controller.js

```diff
@@ -67,6 +67,43 @@ module.exports = function(Chart) {
 		return position === 'top' || position === 'bottom';
 	}
 
+	/**
+	 * Return the y data of the label, if no this label, return null
+	 * @private
+	 * @param {string} label the label in the labels array
+	 * @param {array} dataArray the data in the datasets.
+	 * @return {number} the y data according to the label
+	 */
+	function getY(label, dataArray) {
+		var y = null;
+		for (var i = 0; i < dataArray.length; i++) {
+			var item = dataArray[i];
+			if (item.x === label) {
+				y = item.y;
```

Comment:
you can just `return item.y` here, which will be less code and more efficient. then remove the `y` variable and `return null` at the end if you didn't hit this

Reply:
The code is changed as you required.